### ISSUE TITLE
A crash occurs when you start and cancel a download a couple times behind each other

### ISSUE THREAD

**Issue by MrAndre96** [Bionus/imgbrd-grabber#1394]
**Steps to reproduce**

Enable the source sankaku complex and some other sources like gelbooru and konachan for example
Now search nekopara and click on get all
Now first click on 'Download selected' by sankaku complex
After that click directly on 'Cancel' and now repeat it with just the other sources
For me it mosty takes a couple sources before grabber crashes

(Found this while trying to find the other crashxd)

**System information**

Windows 10 with the latest nightly version

**Comment by MrAndre96** [comment]
Why was the milestone v6.0.5 removed from this crash report?

**Comment by Bionus** [comment]
Because I can't guarantee fixing this bug before releasing 6.0.5, as the source of this crash is quite complex and might need some work to properly fix 😞

Given I planned to release 6.0.5 today (but haven't because of unforeseen circumstances, so it'll be next week), I don't think this fix can be shipped with it. It'll be in 6.0.6.

But aren't you using nightly anyway? 😅

**Comment by MrAndre96** [comment]
Yeah ofc:) But I didn't want that the found crashes would be pushed aside

**Comment by Bionus** [comment]
As long as they're reproducible, they won't be 😉
